RailwayCo/Assets/Scripts/Unity/Tooltip/Tooltip.cs

```diff
@@ -0,0 +1,64 @@
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+[ExecuteInEditMode()]
+public class Tooltip : MonoBehaviour
+{
+    public TextMeshProUGUI headerField;
+    public TextMeshProUGUI contentField;
+    [SerializeField] private LayoutElement layoutElement;
+    [SerializeField] private int characterWrapLimit;
+    private RectTransform _rectTransform;
+
+    private void Awake()
+    {
+        _rectTransform = GetComponent<RectTransform>();
```

Comment:
Add checks to make sure that the SerializeField and public items have something associated (headerField, contentField, layoutElement)

Reply:
Fixed in c9406c3